src/macros.jl

```diff
@@ -231,7 +231,7 @@ function extract_kwargs(args)
     return flat_args, kwargs, requestedcontainer
 end
 
-function addkwargs!(call, kwargs)
+function add_keyword_arguments(call, kwargs)
```

Comment:
(optional) kw seems fine as an abbreviation of keyword

Reply:
This function could use documentation, especially since it modifies the call argument.